ext/redhat/pxp-agent.init

```diff
@@ -46,15 +46,10 @@ start() {
 
 stop() {
     echo -n $"Stopping PXP agent: "
-    if [ "$USEINITFUNCTIONS" = "true" ]; then
-        killproc $pidopts $exec
-        RETVAL=$?
-    else
-        if [ -n "${pid}" ]; then
-            kill -TERM $pid >/dev/null 2>&1
-            RETVAL=$?
-        fi
-    fi
+    trap '' TERM
+    killall $prog 2>/dev/null
```

Comment:
I would lean towards still restricting this to the current user. So
```
killall -u `whoami` $prog 2>/dev/null
```

Reply:
Ah sorry - I was using that on the system I was tsting on but didn't seem to include it in the PR! Good catch :)